modules/text2vec-contextionary/graphql/near_text_argument.go

```diff
@@ -19,7 +19,7 @@ import (
 )
 
 func nearTextArgument(prefix, className string) *graphql.ArgumentConfig {
-	prefixName := fmt.Sprintf("%s%s", prefix, className)
+	prefixName := fmt.Sprintf("Txt2VecC11y%s%s", prefix, className)
```

Comment:
should we make this automatic based on the module name? With the few modules we have at the moment, there is very little risk of a conflict. But I think in the future it might be nice if this was automatically derived from the module name which is already guaranteed to be unique. Maybe create a follow up issue for this?

Reply:
for this change, I think we would then need to change the interfaces:
from:
`GetArguments(classname string) map[string]*graphql.ArgumentConfig`
`ExploreArguments() map[string]*graphql.ArgumentConfig`
to:
`GetArguments(moduleName string, classname string) map[string]*graphql.ArgumentConfig`
`ExploreArguments(moduleName string) map[string]*graphql.ArgumentConfig`

Reply:
Maybe just create a follow up ticket for now? Don't think we have to solve this now. A potential breaking change to the module API is also not an issue as long as we're still in control of all modules.

Reply:
sure! I'll do that

Reply:
https://github.com/semi-technologies/weaviate/issues/1467